src/main/java/com/google/sps/data/ProcessPromotionQuizResults.java

```diff
@@ -0,0 +1,22 @@
+package com.google.sps.data;
+
+import java.util.List;
+
+/** Returns the recommended career path based off the career quiz results. */
+public class ProcessPromotionQuizResults {
+  private ProcessPromotionQuizResults() {}
+
+  public static Boolean getIsPromotedOrNot(List<QuestionChoice> userChoices, double threshold)
+      throws IllegalArgumentException {
```

Comment:
Include a message in the exception so we know what went wrong

Reply:
Added.